### PR TITLE
My solution

### PR DESCRIPTION
**Pull request by kovaliskoveronika**

### CODE REVIEW

**Review by oleksiikolii — APPROVED** [review]

File: main.py [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+
+    create_actors = [("George", "Klooney"),
+                     ("Kianu", "Reaves"),
+                     ("Scarlett", "Keegan"),
+                     ("Will", "Smith"),
+                     ("Jaden", "Smith"),
+                     ("Scarlett", "Johansson")]
+    for data in create_actors:
+        Actor.objects.create(
+            first_name=data[0],
```
Comment (LiudmylaKulinchenko) [inline main.py:25]: use checklist

File: main.py [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
-    pass
+    create_genre = ["Western",
+                    "Action",
+                    "Dramma"]
+    for name in create_genre:
+        Genre.objects.create(
+            name=name
+        )
+
+    create_actors = [("George", "Klooney"),
```
Comment (LiudmylaKulinchenko) [inline main.py:17]: use consistence indentation throughout the code
You should split it like in 13-15 lines (add a line after [ and before ])

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: main.py [hunk 3]
```diff
@@ -2,6 +2,53 @@
...
+    ]
+
+    checklist = [
+        {"field": "first_name", "index": 0},
+        {"field": "last_name", "index": 1}
+    ]
+
+    for data in create_actors:
+        actor_data = {item["field"]: data[item["index"]] for item in checklist}
+        Actor.objects.create(**actor_data)
```
Comment (YuriiKindrat) [inline main.py:36]: it is too complicated solution, try to simplify this 
could u create a several variables in for loop to unpack ur actors ?

File: main.py [hunk 4]
```diff
@@ -2,6 +2,53 @@
...
+        )
+
+    create_actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
```
Comment (YuriiKindrat) [inline main.py:27]: it is bad name for variables here, could u just name it like genres and actors ?

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

**Pull request closed without merging**